tests/components/homekit/test_type_fans.py

```diff
@@ -203,9 +206,8 @@ def cls():
     assert acc.char_speed.value != 0
 
     await hass.async_add_job(acc.run)
-    assert (
-        acc.speed_mapping.speed_ranges == HomeKitSpeedMapping(speed_list).speed_ranges
-    )
+    assert (acc.speed_mapping.speed_ranges == HomeKitSpeedMapping(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4713809)